### ISSUE TITLE
Add Premium Icon indicator to WPcom and FSE blocks

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#44524]
This is a follow up issue of https://github.com/Automattic/jetpack/issues/16628. This one aims to WPcom and FSE blocks:

- premium-content
- core/video

**Comment by cpapazoglou** [comment]
@retrofox Do you have in mind any other blocks?

**Comment by retrofox** [comment]
> @retrofox Do you have in mind any other blocks?

No, I don't. Although `core/audio` is another block to handle it doesn't change its icon afaik.
